front/app/components/UI/NewBadge/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+// components
+import {
+  Box,
+  BoxProps,
+  fontSizes,
+  colors,
+} from '@citizenlab/cl2-component-library';
+
+// i18n
+import { useIntl } from 'utils/cl-intl';
+import messages from './messages';
+
+// utils
+import { lighten } from 'polished';
+
+const NewBadge = (props: BoxProps) => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <Box
```

Comment:
Did you consider using the `Badge` component that is used here? https://github.com/CitizenLabDotCo/citizenlab/blob/5d2bb71637ddb86c7ec543bf1e251921d1d84990/front/app/containers/Admin/projects/project/offlineInputs/TopBar/index.tsx#L74-L76

Reply:
The `Badge` didn't really look like the design... but indeed I will just reuse the `Badge` for consistency :+1: